views/peeps.erb

```diff
@@ -0,0 +1,5 @@
+<ul>
+  <% @peeps.each do |peep| %>
+    <li><%=peep%></li>
```

Comment:
All the rest of your code is there to fulfil user story 3 (showing the timestamp) -- did you run out of time or did you get blocked?